src/components/DataSubmissions/RadioInput.tsx

```diff
@@ -76,10 +77,12 @@ const StyledFormControlLabel = styled(FormControlLabel)(() => ({
   },
 }));
 
-type Option = {
+export type Option = {
   label: string;
   value: string;
   disabled?: boolean;
+  tooltip?: boolean;
```

Comment:
Optional suggestion, but it seems like `tooltip` may be a redundant prop. Can we infer whether to show a tooltip based on if `tooltipContent` is provided instead?

Reply:
Done